src/app/app.module.ts

```diff
@@ -42,7 +42,7 @@ import {
     NgxMatTimepickerModule
 } from '@angular-material-components/datetime-picker';
 import {MatDatepickerModule} from '@angular/material/datepicker';
-import {HighlightModule} from 'ngx-highlightjs';
+import {HIGHLIGHT_OPTIONS, HighlightModule} from 'ngx-highlightjs';
```

Comment:
This was done to fix and error where the highlights were not being loaded in certain areas of the project.